### PR TITLE
Update Debian installation instructions in README.md

### PR DESCRIPTION
**Pull request by mattock**
Signed-off-by: Samuli Seppänen <samuli@openvpn.net>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +55,21 @@ EL users must first [configure EPEL](http://fedoraproject.org/wiki/EPEL/FAQ#How_
...
+
+	git clone git://github.com/freight-team/freight.git
+
+Then create and install the package - here a .deb:
+
+	cd freight
+	make build
+	sudo dpkg -i freight_0.3.5-1_all.deb
+
+See FPM's documentation for details about the command-line options.
```
Comment (domcleal) [inline README.md]: Not relevant?

File: README.md [hunk 2]
```diff
@@ -43,10 +43,11 @@ Serve `/var/cache/freight` via your favorite web server and install it as an APT
 
 ### From a Debian archive
 
-	echo "deb http://packages.rcrowley.org $(lsb_release -sc) main" | sudo tee /etc/apt/sources.list.d/rcrowley.list
-	sudo wget -O /etc/apt/trusted.gpg.d/rcrowley.gpg http://packages.rcrowley.org/keyring.gpg
-	sudo apt-get update
-	sudo apt-get -y install freight
+	sudo -s
```
Comment (mmoll) [inline README.md]: that line can be ommited, IMHO

File: README.md [hunk 3]
```diff
@@ -54,6 +55,21 @@ EL users must first [configure EPEL](http://fedoraproject.org/wiki/EPEL/FAQ#How_
...
+First [install fpm](https://github.com/jordansissel/fpm). Then clone the freight
+Git repository;
+
+	git clone git://github.com/freight-team/freight.git
+
+Then create and install the package - here a .deb:
+
+	cd freight
+	make build
+	sudo dpkg -i freight_0.3.5-1_all.deb
```
Comment (mmoll) [inline README.md]: -2 then also here

**Comment by mattock** [comment]
I didn't have any strong opionions regarding the "Install from custom-made package" section, so I removed it as requested. It was mostly meant for people interested in repackaging freight and as a reminder for ourselves.

I also changed the apt instructions to conform to the general style of README.md by adding a "sudo" in front of every command. That said, "sudo" is not used consistently through README.md - for example:
```
freight-add ...
freight-cache ...
yum -y install freight
```
We might want to prepend these with "sudo" also, or mention that typically you'd run these commands as root. If you want, I can fix this in this PR.

**Comment by domcleal** [comment]
> I didn't have any strong opionions regarding the "Install from custom-made package" section, so I removed it as requested. It was mostly meant for people interested in repackaging freight and as a reminder for ourselves.

Oh sorry, we might have misunderstood each other.  I was just meaning the line about fpm command line options that I commented on, not the whole section - since when running `make build` there's no way to pass those command line options in.

**Comment by mattock** [comment]
Oh, I see. The part you commented on was a leftover indeed. Shall I add back the "Install from custom package" section?

File: README.md [hunk 4]
```diff
@@ -43,10 +43,19 @@ Serve `/var/cache/freight` via your favorite web server and install it as an APT
 
 ### From a Debian archive
 
-	echo "deb http://packages.rcrowley.org $(lsb_release -sc) main" | sudo tee /etc/apt/sources.list.d/rcrowley.list
-	sudo wget -O /etc/apt/trusted.gpg.d/rcrowley.gpg http://packages.rcrowley.org/keyring.gpg
-	sudo apt-get update
-	sudo apt-get -y install freight
+	wget -O - https://swupdate.openvpn.net/repos/repo-public.gpg|sudo apt-key add -
+	echo "deb http://build.openvpn.net/freight_team $(lsb_release -sc) main" | sudo tee  /etc/apt/sources.list.d/freight.list
+	apt-get update
```
Comment (domcleal) [inline README.md]: keep sudo on these two lines?

**Comment by mattock** [comment]
@domcleal: sudo is only needed after the pipe.

**Comment by domcleal** [comment]
If the instructions are meant to be executed as a non-root user as a whole then they'd be needed on apt-get calls too.

**Comment by mattock** [comment]
You're correct, I noticed that right after sending the comment :). Fixing.

**Comment by domcleal** [comment]
:+1:

**Comment by mmoll** [comment]
merged, thanks @mattock

**Comment by mattock** [comment]
No problem! This PR underlined the fact why mandatory review is good. It was surprisingly hard to get even a trivial commit like this just right.

**Pull request merged**